.linthtmlrc.json

```diff
@@ -1,3 +1,49 @@
 {
-  "extends": "@mate-academy/linthtml-config"
+  "attr-bans": [
```

Comment:
![image](https://user-images.githubusercontent.com/38065883/174625115-39b9c41d-3ec9-495c-8511-b5dcc98bbe13.png)
Blocks shouldn't be less than 300px wide